README.md

```diff
@@ -223,7 +223,7 @@ or failure.
 Available options:
 
 + `-n`: Suppresses the normal change of directory when adding directories to the stack, so that only the stack is manipulated.
-+ `-q`: Supresses output to the console.
++ `-q`: Suppresses output to the console.
```

Comment:
Could you please make this change here instead:

https://github.com/shelljs/shelljs/blob/aa9d443e7b8126763cfb76b1230a7bdb578a6a5e/src/dirs.js#L43

re-generate the README with `npm run gendocs`

Reply:
Ya, consider it done :)